content/guides/models/app/features/panels/_index.md

```diff
@@ -61,6 +61,58 @@ To add a custom panel to your workspace:
 
 To learn more about the options for each type of panel, refer to the relevant section below, such as [Line plots]({{< relref "line-plot/" >}}) or [Bar plots]({{< relref "bar-plot.md" >}}).
 
+## Share panels
+
+This section describes the various ways to share a panel and how each way differs.
+
+The settings of a project determine who can view the panel. This means that if the project is private, only members of the project can view the panel. If the project is public, anyone with the link can view the panel.
+
+### View and share a panel in full-screen mode
+In full-screen mode, the panel plots 10,000 buckets rather than 1,000 when not in full-screen mode. The state of the run selector is preserved, so you can toggle runs on or off or search for runs.
+
+1. Hover over the panel, then click the panel's action menu `...`.
+1. Click the full-screen button, which looks like a viewfinder.
+    {{< img src="/images/app_ui/panel_fullscreen.png" alt="View panel full-screen" >}}
+
+    The panel opens in full-screen mode.
+1. Copy the URL from the browser or click the action menu `...` and select **Copy panel URL**.
+1. Share the link with the user or team. When they access the link, the panel opens in full-screen mode automatically.
+
+The proceeding section shows how to determine a panel's full-screen URL programmatically.
+
+#### Compose a panel's full-screen URL programmatically
+In certain situations, such as when [creating an automation]({{< relref "/guides/models/automations/" >}}), it can be useful to include the panel's full-screen URL. This section shows how to compose The URL for a panel programmatically. In the proceeding example, replace the entity, project, panel, and section names in brackets.
+
+```text
+https://wandb.ai/<ENTITY_NAME>/<PROJECT_NAME>?panelDisplayName=<PANEL_NAME>&panelSectionName=<SECTON_NAME>
+```
+
+If multiple panels have the same name, this is the URL for the first panel with the name. In this case, find the URL by [viewing the panel in full-screen mode]({{< relref "#view-and-share-a-panel-in-full-screen-mode" >}}).
```

Comment:
```suggestion
If multiple panels match the specified panel and section names, the first match will be selected.
```

I hesitate to point users back to the prev section, because the URL you get from "Copy panel URL" in the UI may still have the same problem (uses panel/section name query params that have "choose first" conflict resolution)